### ISSUE TITLE
Error while building intrinsics: CameraInit failed with error code 3221226505.

### ISSUE THREAD

**Issue by CommunityJoshua** [alicevision/AliceVision#1411]
I am having trouble importing pictures into Meshroom 2023.1.0 with AliceVision 3.0.0. When I try to import the images, a message box pops up indicating "You probably have a corrupted image within the images that you are trying to import.". In the log, I found the following error code: 3221226505. I am not sure what this means or how to fix the issue. The pictures were taken from my OnePlus 9 Pro, and I have checked them but cannot find any issues. Can anyone help me resolve this problem? It would be great if there is a lookup table available for the error code. 

```
[2023-04-13 14:31:12.353487] [0x00006f70] [trace]   Embedded OCIO configuration file: 'C:\Users\joshu\Downloads\Meshroom-2023.1.0-win64\Meshroom-2023.1.0\aliceVision/share/aliceVision/config.ocio' found.
[2023-04-13 14:32:29.088487] [0x00007344] [trace]   Embedded OCIO configuration file: 'C:\Users\joshu\Downloads\Meshroom-2023.1.0-win64\Meshroom-2023.1.0\aliceVision/share/aliceVision/config.ocio' found.
Program called with the following parameters:
 * allowSingleView = 1
 * allowedCameraModels = "pinhole,radial1,radial3,brown,fisheye4,fisheye1,3deanamorphic4,3deradial4,3declassicld"
 * colorProfileDatabase = "" (default)
 * defaultCameraModel = "" (default)
 * defaultFieldOfView = 45
 * defaultFocalLength = -1 (default)
 * defaultFocalRatio = 1 (default)
 * defaultOffsetX = 0 (default)
 * defaultOffsetY = 0 (default)
 * errorOnMissingColorProfile = 1 (default)
 * groupCameraFallback =  Unknown Type "enum EGroupCameraFallback"
 * imageFolder = "" (default)
 * input = "C:\Users\joshu\AppData\Local\Temp\tmpzsuazzgz/CameraInit/ee5aca9875d6756077ac43f2ac3f4b6023cde933/viewpoints.sfm"
 * lensCorrectionProfileInfo = "${ALICEVISION_LENS_PROFILE_INFO}"
 * lensCorrectionProfileSearchIgnoreCameraModel = 1
 * maxCoresAvailable =  Unknown Type "unsigned int" (default)
 * maxMemoryAvailable = 18446744073709551615 (default)
 * output = "C:/Users/joshu/AppData/Local/Temp/tmpzsuazzgz/CameraInit/ee5aca9875d6756077ac43f2ac3f4b6023cde933/cameraInit.sfm"
 * rawColorInterpretation =  Unknown Type "enum aliceVision::image::ERawColorInterpretation"
 * sensorDatabase = "C:\Users\joshu\Downloads\Meshroom-2023.1.0-win64\Meshroom-2023.1.0\aliceVision\share\aliceVision\cameraSensors.db"
 * verboseLevel = "info"
 * viewIdMethod =  Unknown Type "enum aliceVision::sfmDataIO::EViewIdMethod"
 * viewIdRegex = ".*?(\d+)" (default)

Hardware :
        Detected core count : 24
        OpenMP will use 24 cores
        Detected available memory : 13435 Mo

[14:32:29.121487][info] 0 profile(s) stored in the LCP database.
ERROR:root:Error while building intrinsics: CameraInit failed with error code 3221226505.
Command was: "aliceVision_cameraInit  --sensorDatabase "C:\Users\joshu\Downloads\Meshroom-2023.1.0-win64\Meshroom-2023.1.0\aliceVision\share\aliceVision\cameraSensors.db" --lensCorrectionProfileInfo "${ALICEVISION_LENS_PROFILE_INFO}" --lensCorrectionProfileSearchIgnoreCameraModel True --defaultFieldOfView 45.0 --groupCameraFallback folder --allowedCameraModels pinhole,radial1,radial3,brown,fisheye4,fisheye1,3deanamorphic4,3deradial4,3declassicld --rawColorInterpretation LibRawWhiteBalancing --viewIdMethod metadata --verboseLevel info --output "C:/Users/joshu/AppData/Local/Temp/tmpzsuazzgz/CameraInit/ee5aca9875d6756077ac43f2ac3f4b6023cde933/cameraInit.sfm" --allowSingleView 1 --input "C:\Users\joshu\AppData\Local\Temp\tmpzsuazzgz/CameraInit/ee5aca9875d6756077ac43f2ac3f4b6023cde933/viewpoints.sfm"".

WARNING:root:QQmlComponent: Created graphical object was not placed in the graphics scene.
```

**Comment by CommunityJoshua** [comment]
![IMG20230413132935](https://user-images.githubusercontent.com/21293543/231762010-a8af22e4-8be9-4fc2-8ec5-e9b9b7c9d8d3.jpg)

**Comment by CommunityJoshua** [comment]
I was able to successfully import the "defective" image i just uploaded into Meshroom. However, I'm not sure why or what fixed it. Bad Peggy 2.4.0 did not detect any errors. Batch converting with IrfanView did not work, even with a quality lower than 100%.

**Comment by Jojo-1000** [comment]
@CommunityJoshua Thank you! I was able to fix my file imports just by adding 'a' to the filename

**Comment by Jojo-1000** [comment]
From what I saw in the code, there seems to be an image sequence detection in the CameraInit:
https://github.com/alicevision/AliceVision/blob/9463f9503f29bca301c0935078c581d828fdf481/src/software/pipeline/main_cameraInit.cpp#L454-L463

Maybe this is what causes the error with numeric filenames? If that is the case, this should be reworked, because all of my camera images have numbers by default. Also there is supposed to be an error log, but it is in a temp file which is immediately deleted by meshroom, so I cannot look at it

**Comment by cbentejac** [comment]
Hi @Jojo-1000, thanks for reporting and identifying the issue! It was indeed caused by the `std::stoi` you pointed out.

This will be fixed with #1565.
